src/test/regress/expected/multi_colocation_utils.out

```diff
@@ -1037,8 +1037,8 @@ SELECT create_distributed_table('table1_groupG', 'id');
 UPDATE pg_dist_partition SET repmodel = 's' WHERE logicalrelid = 'table1_groupG'::regclass;
 CREATE TABLE table2_groupG ( id int );
 SELECT create_distributed_table('table2_groupG', 'id', colocate_with => 'table1_groupG');
-ERROR:  cannot colocate tables table1_groupg and table2_groupg
-DETAIL:  Replication models don't match for table1_groupg and table2_groupg.
+ERROR:  replication factors above one are incompatible with the streaming replication model
```

Comment:
This error is a little surprising. I would expect colocate_with to override other settings.

Reply:
The error message might be misleading. Here is the actual reason why we error out here:

- Streaming replication works only when the shard replication factor is 1
- When the colocation option is supplied, `create_distributed_table` uses the same replication model as the colocated table (streaming in this case)
- Since we updated the replication model of a table with replication factor 2 to use streaming replication, we need to error out since streaming replication does not allow this replication factor

In a real life scenario I can not think of a way to hit this error here, as the replication model should never be streaming when replication factor > 1.